wolfcrypt/test/test.c

```diff
@@ -5034,6 +5034,13 @@ int chacha20_poly1305_aead_test(void)
     byte generatedPlaintext[272];
```

Comment:
A comment would be useful as to why this magic number 272 is used (max of the input (264) is aligned to the next 16 bytes)?

Reply:
This was existing code and I agree, bad use of size without comment. Adjusted to match required max and added comment. I suspect you are right about the 16-round up, but there is no requirement for this in ChaCha20 or Poly1305 code.